sdk/servicebus/azure-messaging-servicebus/CHANGELOG.md

```diff
@@ -8,6 +8,8 @@
 
 ### Bugs Fixed
 
+- Where SynchronousMessageSubscriber would not receive additional messages if they were > 1 and the timeout had elapsed for the first receive call. https://github.com/Azure/azure-sdk-for-java/issues/25063
```

Comment:
Can we change this to 

"Fixed a bug where Synchronous Receiver client stops receiving messages if MaxMessages is greater than 1.  Issue [25063]"

Reply:
I like the re-wording Josh is suggesting here, but would recommend to keep the part about the timeout because in happy paths count > 1 works just fine